sys/include/net/ipv6/addr.h

```diff
@@ -669,6 +669,55 @@ char *ipv6_addr_to_str(char *result, const ipv6_addr_t *addr, uint8_t result_len
  */
 ipv6_addr_t *ipv6_addr_from_str(ipv6_addr_t *result, const char *addr);
 
+/**
+ * @brief split IPv6 address string representation
+ *
+ * @note Will change seperator position in @p addr to '\0'
+ *
+ * @param[in]   addr        Address to split
```

Comment:
to match the signature: rename `addr` to `addr_str` here and in the `@note` section as well